### ISSUE TITLE
unable to load runtime environment

### ISSUE THREAD

**Issue by awright424** [Jorl17/jar2app#4]
I have been trying to figure out how to get around this error when your using your app. I am able to get it to output the jar but receive an error when attempting to run it. I have also signed the application as well to only have it result in the same error.

**Comment by awright424** [comment]
`jar2app UserProxy.jar rockit -r /Library/Java/JavaVirtualMachines/jdk1.8.0_111.jdk`

that is the command used. Any help would be appreciated. this is also running on osx sierra 10.12.1

**Comment by Jorl17** [comment]
Hi!

I'm hoping that this is fixed by https://github.com/Jorl17/jar2app/commit/870c28c026f4eb86003681c8e2eaed6f85af1abe . Can you pull and try again? Thanks :)

**Comment by awright424** [comment]
I no longer receive said error but after signing and moving to my applications folder it would seem something is not configured to be relative but to depend on absolute path

**Comment by Jorl17** [comment]
How so? Can you perhaps provide a more specific example?

**Comment by awright424** [comment]
LSOpenURLsWithRole() failed with error -10810 for the URLs

this is my error now so I believe your job is done here. I need to remember what I had done a year ago to get past this. thanks

**Comment by Jorl17** [comment]
Ah, right, I thought it was still something in jar2app. Well, thank you for your bug report! Good luck :)

**Comment by forresthopkinsa** [comment]
I'm also encountering this error right now. Hard to debug as there is very little error data logged.

`jar2app app.jar -i app.icns -r ~/.sdkman/candidates/java/8.0.192-zulu`

**Comment by Lucina** [comment]
From what I understand (given the 8u192 build I've downloaded), zulu builds are not in the Apple bundle structure (`Contents/Home/<bin, db, include...>`) and instead just have bin/db/etc directly under the root jdk directory. The included JavaAppLauncher doesn't support that. I get the error as well when I tried using the build I have. You'll need to use a different jdk that's packaged as the stub expects (like standard library installs at `/Library/Java/JavaVirtualMachines` for example) or find another stub that supports it.
